modules/mcc/include/opencv2/mcc/checker_model.hpp

```diff
@@ -89,6 +89,15 @@ class CV_EXPORTS_W CChecker
 
     CV_WRAP virtual TYPECHART getTarget() = 0;
     CV_WRAP virtual std::vector<Point2f> getBox() = 0;
+
+    /** @brief Computes and returns the coordinates of the central parts of the charts modules.
+     *
+     * This method computes transformation matrix from the checkers's coordinates (`cv::mcc::CChecker::getBox()`)
+     * and find by this the coordinates of the central parts of the charts modules.
+     * It is used in `cv::mcc::CCheckerDraw::draw()` and in `ChartsRGB` calculation.
+     */
+    CV_WRAP virtual std::vector<Point2f> getComputedCharts() = 0;
```

Comment:
M.b. `getColorCharts()` is more relevant.

Reply:
Mb `getComputedColorCharts()`?

Reply:
I propose to name it `getColorCharts` without `computed`. The function behaviour may change.

Reply:
fixed